setup.py

```diff
@@ -275,8 +275,8 @@ def get_version_info():
 
     # If this is a release or another kind of source distribution of PyCBC
     except:
-        version = '1.9.1dev'
-        release = 'False'
+        version = '1.9.2'
```

Comment:
Sorry, I didn't really explain versioning well. This is fine, but we haven't released 1.9.1 yet, so you should probably set it to that, and when you then revert the version to a non-release, set it to 1.9.2dev (i.e what will later become the 1.9.2 release).

Reply:
My bad, I was thinking that we had made a 1.9.1 [failed] release, but apparently I made that up.